provider/source/Cargo.toml

```diff
@@ -56,6 +56,7 @@ elsa = { workspace = true }
 itertools = { workspace = true }
 log = { workspace = true }
 ndarray = { workspace = true }
+parse-zoneinfo = { workspace = true }
```

Comment:
Question: why did you switch from `jiff_tzdb` + `tzif` to `parse-zoneinfo`?

We know the maintainers of the first two crates, and of course `tzif` lives in this repository.

Reply:
The TZDB is distributed in text format at https://github.com/eggert/tz. It is not distributed anywhere we could find in binary format (#). Using `jiff_tzdb` means Burntsushi has to update their crate every time a new tzdb release happens, and users then have to recompile `icu4x-datagen`.

Reply:
The files can also be found at https://data.iana.org/time-zones/. But to get the binary formats `zic.c` would need to be implemented in Rust or some other alternative.